Clever Mutation

Sneeker134 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Basically, the Pull Request just adds a new trait that allows for mobs that might otherwise not be able to do stuff like use computers, do surgery or interact with PDAs to do so. Primarily aimed at monkeys, though there may be other mobs that this could be useful to as well.

## Why It's Good For The Game

Being a monkey is kind of fun, but unfortunately for any job that requires you to work a console, you can't do it. This includes the person most likely to actually become a monkey, the geneticist. It's not like being a monkey doesn't come with some pretty big downsides anyway, so adding this seems like a pretty simple way to give them a bit more game. I doubt you'll really see many more people try to be a monkey, but for the few who do hopefully this way they might have a bit more fun.

## Changelog


:cl:
add: New clever mutation, allowing some mobs such as monkeys to use consoles.
/:cl:

Sneeker134 committed:
Added new mutation

Basically, just gives you a way to allow stuff like a monkey to access consoles and shit.

JohnFulpWillard commented on code/datums/mutations/passive.dm:
```diff
@@ -11,3 +11,19 @@
+	name = "Clever"
+	desc = "Causes the subject to feel just a little bit smarter."
+	quality = POSITIVE
+	instability = 5
```
for something meant to give a massive boost to monkeys, this really doesn't give the 'good gene' drawback of a lot of instability. I expected at least 20 imo.

Sneeker134 replied:
Yeah, that seems fair enough. I felt that since it's something that's basically never done, a big power boost to it isn't really a big deal. Just as a raw utility boost for what it does (even if it doesn't do anything most of the time), it is pretty good. I'll up the instability some.

Sneeker134 committed:
Made Clever More Unstable

It does add quite a bit of lot in the few situations its actually useful, so having a higher instability makes some sense.

B4CKU commented:
i absolutely love this, but i feel like it'd be nice to get a message whenever you gain/lose this mutation. maybe try using `text_lose_indication` and `text_gain_indication`?

Pickle-Coding commented:
Not a big fan of being able to counter the downsides of a species with a mutation, because then they're not really downsides anymore.

gonenoculer5 commented:
#66650 Devils PR.

Sneeker134 committed:
Adds text on gaining and losing

QoL for using it, just so you can make sure you have the mutation before you transform into whatever you were gonna transform into.

Sneeker134 commented:
> i absolutely love this, but i feel like it'd be nice to get a message whenever you gain/lose this mutation. maybe try using `text_lose_indication` and `text_gain_indication`?

Good idea, added it in.

moocowswag commented:
I would up the instability a little bit, since monkeys cant be hulks and have no use for shock touch they dont have a lot of things to budget their instability on.

Sneeker134 committed:
Changed description

Basically just adding a little hint in that this should be used on "lesser" species.

jughu commented:
>have a mutation that disables a aspect of the game
>now add a mutation that removes that aspect limitation

tralezab commented:
> Not a big fan of being able to counter the downsides of a species with a mutation, because then they're not really downsides anymore.

I actually agree with this but I really think genetics is near rock bottom when it comes to what you get from it, and nobody has embarked on fixing it so for the moment this seems like an innocent way to breathe life into the job until it is properly reworked. (As a refresher nothing too strong can be offered in genetics because it is too easy to spread powers around without much input from the geneticist)

tralezab commented:
(get yo gbp up king)
